scripts/tftp-version.nse

```diff
@@ -0,0 +1,331 @@
+local nmap = require "nmap"
+local stdnse = require "stdnse"
+local string = require "string"
+local shortport = require "shortport"
+local table = require "table"
+
+description=[[
+Obtains information (such as vendor and device type where available) from a
+TFTP service. Software vendor information is deduced based on error messages.
+]]
+
+author = "Mak Kolybabi <mak@kolybabi.com>"
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+categories = {"default", "discovery", "safe", "version"}
```

Comment:
I don't think "discovery" is appropriate, because we're not discovering anything, only determining version, which is already covered by "version." Will have to think about "default" and "safe" vs "intrusive" because we're requesting a nonexistent file. How big of a deal is that? I'd guess not many places check TFTP logs, but do you have a better idea?